### ISSUE TITLE
Term 2 - Lesson 2 - 30. Keras - Small typo in text

### ISSUE THREAD

**Issue by miguelusque** [udacity/aind-issue-reports#151]

**Comment by miguelusque** [comment]
Hi!

I have noticed a small typo at Term 2, lesson 2, 30. keras text.

This is the existing code (please noticed I have highlighted in bold then typos):
>   \# 1st Layer - Add an input layer of 32 nodes with the same input shape as
>   \# the training samples in X
>    model.add(Dense(32, input_dim=X.shape[1]))
>
>    \# __2rd__ Layer - Add a softmax activation layer
>    model.add(Activation('softmax'))

>    \# __4th__ Layer - Add a fully connected output layer
>    model.add(Dense(1))

>    \# __5th__ Layer - Add a sigmoid activation layer
>    model.add(Activation('sigmoid'))

I think it would be more accurate like this:
>   \# 1st Layer - Add an input layer of 32 nodes with the same input shape as
>   \# the training samples in X
>    model.add(Dense(32, input_dim=X.shape[1]))
>
>    \# __2nd__ Layer - Add a softmax activation layer
>    model.add(Activation('softmax'))

>    \# __3rd__ Layer - Add a fully connected output layer
>    model.add(Dense(1))

>    \# __4th__ Layer - Add a sigmoid activation layer
>    model.add(Activation('sigmoid'))

I hope it helps!

BR

Miguel Ángel

**Comment by luisguiserrano** [comment]
Thank you! Good catch. Fixed.
